firestore/src/ios/firestore_ios.cc

```diff
@@ -299,9 +322,74 @@ void Firestore::set_log_level(LogLevel log_level) {
                                                     : log_level);
 }
 
-void FirestoreInternal::SetClientLanguage(const std::string& language_token) {
+void FirestoreInternal::SetClientLanguage(const std::string &language_token) {
   api::Firestore::SetClientLanguage(language_token);
 }
 
+Future<LoadBundleTaskProgress> FirestoreInternal::LoadBundle(
+    const std::string &bundle) {
+  auto promise = promise_factory_.CreatePromise<LoadBundleTaskProgress>(
+      AsyncApi::kLoadBundle);
+  auto bundle_stream = absl::make_unique<util::ByteStreamCpp>(
+      absl::make_unique<std::stringstream>(std::stringstream(bundle)));
```

Comment:
Is it necessary to create the temporary `stringstream` object for the call to `make_unique()`? Could you just specify `bundle` directly? Also on line 358.

Reply:
Done.